src/Controller/MenuController.java

```diff
@@ -149,6 +153,7 @@
 	@FXML
 	private Button calendar;
 	@FXML
+	private Button chat;
 	private Button closeDrawer;
```

Comment:
It looks like you added the `chat` button here, but sort of took over the `@FXML` annotation which belonged to `closeDrawer`.  Both the `closeDrawer` button and `chat` button are referred to in `MainMenu.fxml`, so they will both need the `@FXML` annotation.  Please add the missing annotation here.